propulsion-reactor/Program.fs

```diff
@@ -545,7 +579,7 @@ let build (args : Args.Arguments) =
         let stats = Ingester.Stats(Log.Logger, statsInterval = TimeSpan.FromMinutes 1., stateInterval = TimeSpan.FromMinutes 5.)
         let sink = Propulsion.Streams.StreamsProjector.Start(Log.Logger, args.MaxReadAhead, args.MaxConcurrentStreams, handle, stats = stats)
 #endif // !kafka
-        let connect () = let c = connectEs () in c.ReadConnection
+        let connect () = connectProjEs
```

Comment:
waaait... its supposed to generate a new one per call, i.e. extra `()` needed on L533 (and then probably reference `connectProjEs` directly instead of aliasing here )

Reply:
Yes, good catch.

Reply:
looks good 👍